webserver/src/main/java/io/kestra/webserver/utils/RequestUtils.java

```diff
@@ -13,14 +14,14 @@ public static Map<String, String> toMap(List<String> queryString) {
         return queryString == null ? null : queryString
             .stream()
             .map(s -> {
-                String[] split = s.split(":");
-                if (split.length != 2) {
+                String[] split = s.split("[: ]+");
+                if (split.length < 2) {
                     throw new HttpStatusException(HttpStatus.UNPROCESSABLE_ENTITY, "Invalid queryString parameter");
                 }
 
                 return new AbstractMap.SimpleEntry<>(
                     split[0],
-                    split[1]
+                    String.join(":", Arrays.copyOfRange(split, 1, split.length))
```

Comment:
This is the same as using indexOf(':') and taking what' s before as key and what's after as value right?

Reply:
By doing that we miss some protection that we have before with miss-crafted query string

Reply:
I don't think we should handle the syntax at this level. The goal of this method should only be to extract before & after first ":" imo.
For the indexOf yes I can also do that :+1:

Reply:
So in this case indexOf is straightforward